tests/selenium/test_sizing.py

```diff
@@ -174,49 +215,10 @@ def update_styles(n_clicks):
 
 
 def test_szng002_percentages_result_in_same_widths(test):
-    base_props = dict(
-        columns=[
-            {"id": "_", "name": ["_", "_", "_"]},
-            {
-                "id": "a",
-                "name": [
-                    "A-----------------VERY LONG",
-                    "A-----------------VERY LONG",
-                    "A-----------------VERY LONG",
-                ],
-            },
-            {
-                "id": "b",
-                "name": [
-                    "A-----------------VERY LONG",
-                    "A-----------------VERY LONG",
-                    "B-----------------VERY LONG",
-                ],
-            },
-            {
-                "id": "c",
-                "name": [
-                    "A-----------------VERY LONG---",
-                    "B-----------------VERY LONG---------",
-                    "C-----------------VERY LONG------------------",
-                ],
-            },
-        ],
-        data=[
-            {"_": 0, "a": 85, "b": 601, "c": 891},
-            {"_": 0, "a": 967, "b": 189, "c": 514},
-            {"_": 0, "a": 398, "b": 262, "c": 743},
-            {"_": 0, "a": 89, "b": 560, "c": 582},
-            {"_": 0, "a": 809, "b": 591, "c": 511},
-        ],
-        style_table=dict(width=500, minWidth=500, maxWidth=500, paddingBottom=10),
-        style_cell=dict(width="25%", minWidth="25%", maxWidth="25%"),
-    )
+    global base_props
```

Comment:
you aren't modifying `base_props` right? Then you don't need this declaration.

Reply:
Oh, nice. I thought it was required just to use it :)

Reply:
https://github.com/plotly/dash-table/pull/798/commits/c93050c95940ba127f784c9a731a4c8ed7ab44e6